www/source/partials/docs/_using-hab-config-updates.html.md.erb

```diff
@@ -16,7 +16,7 @@ You can supply a `user.toml` containing any configuration data that you want to
 Override default configuration data through the use of an environment variable with the following format: `HAB_PACKAGENAME='{"keyname1":"newvalue1", "tablename1":{"keyname2":"newvalue2"}}'`.
 
 ```shell
-$ HAB_MYTUTORIALAPP='{"message":"Habitat rocks!"}' hab start <origin>/<packagename>
+$ HAB_MYTUTORIALAPP='{"message":"Habitat rocks!"}' hab svc start <origin>/<packagename>
```

Comment:
This (and the next one) are ok. They've already been updated to `hab run`. See https://www.habitat.sh/docs/using-habitat/